lib/pluginmanager/update.rb

```diff
@@ -53,9 +53,8 @@ def update_gems!
 
     # any errors will be logged to $stderr by invoke!
     # Bundler cannot update and clean gems in one operation so we have to call the CLI twice.
-    output = LogStash::Bundler.invoke!(:update => plugins)
+    output = LogStash::Bundler.invoke!(:update => plugins, :rubygems_source => gemfile.gemset.sources)
```

Comment:
Some comment, the install doesn't need to know the concept of the rubygems source.